### ISSUE TITLE
(feat request) Generate query string only

### ISSUE THREAD

**Issue by rosostolato** [jin-qu/jinqu-odata#19]
First I want to thanks for this awesome lib. 

My request is that I could generate the query string only without making a http request. I have my own implementation of http client using angular, I know that I can use a custom provider but it would be better if I could only get a query string like this:

```
const query = service(myModel).where(x => x.id > 5).toString();
myService.getList(query).subscribe();
```

**Comment by umutozel** [comment]
I'm glad you like it.

That's a really good idea, it will be useful for testing too. Will be looking into it.

**Comment by umutozel** [comment]
Implemented with 0.3.3.
